### DISCUSSION TITLE
Have Renovate not even look up certain dependencies

### DISCUSSION THREAD

**Discussion by knechtionscoding** [renovatebot/renovate#22548] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

whitesoure/renovate:2.6.0

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I want to completely ignore unknown dependencies from the kubernetes manager. Currently Warnings are thrown in the dependency dashboard when it tries to do a lookup on resources that aren't in the manually curated list. I'd like to tell renovate to completely ignore those and not throw a warning on them because we know they can't be looked up.

### What you tried so far.

In order to not have to disable each specific other type of k8s resource, we tried the inverse to disable everything else and then only enable ones listed in the manually curated list. That hasn't worked.

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    ":dependencyDashboard",
    ":semanticPrefixFixDepsChoreOthers",
    ":ignoreModulesAndTests",
    ":autodetectPinVersions",
    ":prHourlyLimitNone",
    ":prConcurrentLimitNone",
    ":preserveSemverRanges",
    "group:monorepos",
    "group:recommended",
    "workarounds:all",
    "github>whitesource/merge-confidence:beta"
  ],
  "enabledManagers": [
    "kubernetes"
  ],
  "packageRules": [
    {
      "groupName": "kubernetes-enabled",
      "matchManagers": [
        "kubernetes"
      ],
      "matchPackagePatterns": [
        ".*APIService.*",
        ".*CertificateSigningRequest.*",
        ".*ClusterRole.*",
        ".*ClusterRoleBinding.*",
        ".*CronJob.*",
        ".*CSIDriver.*",
        ".*CSINode.*",
        ".*CustomResourceDefinition.*",
        ".*DaemonSet.*",
        ".*Deployment.*",
        ".*EndpointSlice.*",
        ".*Event.*",
        ".*FlowSchema.*",
        ".*HorizontalPodAutoscaler.*",
        ".*Ingress.*",
        ".*IngressClass.*",
        ".*Lease.*",
        ".*LocalSubjectAccessReview.*",
        ".*MutatingWebhookConfiguration.*",
        ".*NetworkPolicy.*",
        ".*PodDisruptionBudget.*",
        ".*PodSecurityPolicy.*",
        ".*PriorityClass.*",
        ".*PriorityLevelConfiguration.*",
        ".*ReplicaSet.*",
        ".*Role.*",
        ".*RoleBinding.*",
        ".*RuntimeClass.*",
        ".*SelfSubjectAccessReview.*",
        ".*SelfSubjectRulesReview.*",
        ".*StatefulSet.*",
        ".*StorageClass.*",
        ".*SubjectAccessReview.*",
        ".*TokenReview.*",
        ".*ValidatingWebhookConfiguration.*",
        ".*VolumeAttachment.*"
      ]
    },
    {
      "groupName": "kubernetes-disable",
      "matchManagers": [
        "kubernetes"
      ],
      "matchDepTypes": [
        "kubernetes-api"
      ],
      "enabled": false
    }
  ],
  "kubernetes": {
    "fileMatch": [
      "\\.yaml$"
    ]
  }
}

```

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
You can add dependencies to `ignoreDeps`, or you can use `packageRules` matchPackageNames or matchPackagePatterns plus `enabled=false`. If packageRules aren't working then something about your match* must be wrong.

**Reply by viceice** [reply]
rule order is important, last rule wins, so your second rule will disable all
